apps/codeforafrica/src/lib/data/common/index.js

```diff
@@ -1,8 +1,10 @@
 import getNavigation from "@/codeforafrica/lib/data/common/navigation";
 
 export async function getPageProps(api) {
+  const footer = await api.findGlobal("footer");
   const navigation = await getNavigation(api);
```

Comment:
Lets be consistent: If we define functions such as getNavigation, then lets define for all. Otherwise, lets call api directly for all.